apps/examples/hello_tfmicro/main.cc

```diff
@@ -0,0 +1,158 @@
+	error_reporter->Report("x_value: %f, y_value: %f\n", x_value, y_value);
+}
+
+// The name of this function is important for Arduino compatibility.
+void setup()
+{
+	// Set up logging. Google style is to avoid globals or statics because of
+	// lifetime uncertainty, but since this has a trivial destructor it's okay.
+	// NOLINTNEXTLINE(runtime-global-variables)
+	static tflite::MicroErrorReporter micro_error_reporter;
+	error_reporter = &micro_error_reporter;
+
+	// Map the model into a usable data structure. This doesn't involve any
+	// copying or parsing, it's a very lightweight operation.
+	model = tflite::GetModel(g_sine_model_data);
+	if (model->version() != TFLITE_SCHEMA_VERSION) {
+		error_reporter->Report(
+			"Model provided is schema version %d not equal "
+			"to supported version %d.",
+			model->version(), TFLITE_SCHEMA_VERSION);
+		return;
+	}
+
+	// This pulls in all the operation implementations we need.
+	// NOLINTNEXTLINE(runtime-global-variables)
+	static tflite::ops::micro::AllOpsResolver resolver;
+
+	// Build an interpreter to run the model with.
+	static tflite::MicroInterpreter static_interpreter(
+		model, resolver, tensor_arena, kTensorArenaSize, error_reporter);
+	interpreter = &static_interpreter;
+
+	// Allocate memory from the tensor_arena for the model's tensors.
+	TfLiteStatus allocate_status = interpreter->AllocateTensors();
+	if (allocate_status != kTfLiteOk) {
+		error_reporter->Report("AllocateTensors() failed");
+		return;
+	}
+
+	// Obtain pointers to the model's input and output tensors.
+	input = interpreter->input(0);
+	output = interpreter->output(0);
+
+	// Keep track of how many inferences we have performed.
+	inference_count = 0;
+}
+
+// The name of this function is important for Arduino compatibility.
+void loop()
+{
+	// Calculate an x value to feed into the model. We compare the current
+	// inference_count to the number of inferences per cycle to determine
+	// our position within the range of possible x values the model was
+	// trained on, and use this to calculate a value.
+	float position = static_cast<float>(inference_count) /
+					 static_cast<float>(kInferencesPerCycle);
+	float x_val = position * kXrange;
+
+	// Place our calculated x value in the model's input tensor
+	input->data.f[0] = x_val;
+
+	// Run inference, and report any error
+	TfLiteStatus invoke_status = interpreter->Invoke();
+	if (invoke_status != kTfLiteOk) {
+		error_reporter->Report("Invoke failed on x_val: %f\n",
+							   static_cast<double>(x_val));
+		return;
+	}
+
+	// Read the predicted y value from the model's output tensor
+	float y_val = output->data.f[0];
+
+	// Output the results. A custom HandleOutput function can be implemented
+	// for each supported hardware target.
+	HandleOutput(error_reporter, x_val, y_val);
+
+	// Increment the inference_counter, and reset it if we have reached
+	// the total number per cycle
+	inference_count += 1;
+	if (inference_count >= kInferencesPerCycle)
+	{
```

Comment:
Except external, arch and bsp, we should follow TizenRT coding rules. Could you fix violations of coding rules?
```
if ( )
{
->
if ( ) {
```
```
void xx()
{
->
void xx(void)
{
```
spaces -> tabs for indetation